.cmake-format.py

```diff
@@ -1,7 +1,5 @@
-# SPDX-FileCopyrightText: 2024 OGL authors
-#
-# SPDX-License-Identifier: GPL-3.0-or-later
-
+# SPDX-License-Identifier: Unlicense
+# SPDX-FileCopyrightText: 2023 NeoFOAM authors
```

Comment:
Is it because this file comes from the NeoFOAM?

Reply:
Good catch, i'll fix it.